src/expr.rs

```diff
@@ -79,6 +79,17 @@ impl PyExpr {
         })
     }
 
+    /// Returns the name of this expression as it should appear in a schema. This name
+    /// will not include any CAST expressions.
+    fn display_name(&self) -> PyResult<String> {
+        Ok(self.expr.display_name()?)
+    }
+
+    /// Returns a full and complete string representation of this expression.
+    fn canonical_name(&self) -> PyResult<String> {
+        Ok(self.expr.canonical_name())
+    }
```

Comment:
minor nit: this method is infallible, so no need to wrap in a result.

```suggestion
    fn canonical_name(&self) -> String {
        self.expr.canonical_name()
    }
```